web/admin/src/FormView.js

```diff
@@ -502,6 +572,87 @@ formInput = (title, name, value) => {
                   value={this.props.newCell.image4}
                   onChange={this.handleInputChange} />
               </label>
+              <label className="boxTitle">
+                Image 4 Link URL:
+                <input
+                  name="url4"
+                  type="text"
+                  className="box"
+                  required
+                  value={this.props.newCell.url4}
+                  onChange={this.handleInputChange} />
+              </label>
+              {this.footerInfo()}
+              <input type="submit" value="Submit Content" className="formButton" />
+            </form>
+          )
+        }
+
+        if (this.props.newCell.type === "Squares Row"){
+          return(
+            <form className="formBox" onSubmit={this.handleSubmit}>
+              {this.checkQuestions()}
+              {this.headerInfo()}
+              <label className="boxTitle">
+                Image 1:
+                <input
+                  className="box"
+                  name="image1"
+                  type="text"
+                  required
+                  value={this.props.newCell.image1}
+                  onChange={this.handleInputChange} />
+              </label>
+              <label className="boxTitle">
+                Image 1 Link URL:
+                <input
+                  name="url1"
+                  type="text"
+                  className="box"
+                  required
+                  value={this.props.newCell.url1}
+                  onChange={this.handleInputChange} />
+              </label>
+              <label className="boxTitle">
+                Image 2:
+                <input
+                  className="box"
+                  name="image2"
+                  type="text"
+                  required
+                  value={this.props.newCell.image2}
+                  onChange={this.handleInputChange} />
+              </label>
+              <label className="boxTitle">
+                Image 2 Link URL:
+                <input
+                  name="url2"
+                  type="text"
+                  className="box"
+                  required
+                  value={this.props.newCell.url2}
+                  onChange={this.handleInputChange} />
+              </label>
+              <label className="boxTitle">
+                Image 3:
+                <input
+                  className="box"
+                  name="image3"
+                  type="text"
+                  required
+                  value={this.props.newCell.image3}
+                  onChange={this.handleInputChange} />
+              </label>
+              <label className="boxTitle">
+                Image 3 Link URL:
```

Comment:
Check all your copy-pastes for errors like this one.  Better to refactor to avoid these duplications.  Maybe have a prop that is an array of length up to 4, with objects that have an `image` and `url` prop?

Reply:
Whats the specific error in question?

Reply:
Looks like I misread, but aren't there 4 images/URLs allowed?

Reply:
This particular module has 3 images. I may refactor to make it dynamic and allow anywhere from 2-4.